source/geod24/LocalRest.d

```diff
@@ -787,6 +804,20 @@ public final class RemoteAPI (API) : API
             return this.childTid;
         }
 
+        /***********************************************************************
+
+            Send a message to the thread to immediately shut down.
+            This method will block until the child thread shuts down.
+
+        ***********************************************************************/
+
+        public void shutdown () @trusted
+        {
+            import std.exception;
+            C.send(this.childTid, ShutdownCommand(C.thisTid()));
+            enforce(C.receiveTimeout(10.seconds, (ShutdownCommand s) => s));
```

Comment:
That's a looooong timeout. Maybe use the node's timeout when available ?

Reply:
Haha, random choice! Good suggestion.